lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -31,9 +35,48 @@ def configure_node(settings, config)
 
       private
 
+      def configure_ip_display(config, settings)
+        extension_lookup_path = TeracyDev::Util.extension_lookup_path(settings, 'teracy-dev-essential')
+
+        networks = get_all_configured_networks(settings)
+
+        # default VM network (127.0.0.1) is at 0 index
+        # we use the latest defined network IP
+        host_ip_index = networks.length
+
+        # if there are no networks defined
```

Comment:
if there is no "private_network" or "public_network" defined which means no accessible IP, do nothing.

Reply:
It's always have at least one accessible ip on the default interface.

Reply:
so we should use that default interface instead of creating a new private_network interface.